src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Logger/Commands.RecoveryServices.Backup.Logger.Netcore.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@MiYanni any reason that this was the only piece of `RecoveryServices` that was updated?

Reply:
It is the only project that built without issue. The other projects all had issues or relied on each other. The logger is completely independent. You can see them (in strike-through) on the list here: https://github.com/Azure/azure-powershell/issues/5719